src/com/intellij/plugins/bodhi/pmd/filter/VirtualFileFilters.java

```diff
@@ -0,0 +1,42 @@
+package com.intellij.plugins.bodhi.pmd.filter;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.VirtualFileFilter;
+
+public class VirtualFileFilters
+{
+    public static VirtualFileFilter and(VirtualFileFilter... filters)

```

Comment:
These short static methods are nice :+1: . You could move all the other filter classes (like `OrFilter` etc) inside this file since they are not meant to be used directly.